L/libcrossterm/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcrossterm"
+version = v"0.2.0"
+
+sources = [
+  GitSource(
+    "https://github.com/kdheepak/libcrossterm.git",
+    "65b89e0f18fc30e9d4a812327507dd9b468c1597"
+  )
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/libcrossterm/
+
+cargo build --release -j${nproc} --target=${rust_target}
+
+if [[ "${target}" == *-w64-mingw32* ]]; then
+    # Windows generates .dlls without the lib prefix
+    install -Dvm 0755 "target/${rust_target}/release/crossterm.${dlext}" "${libdir}/libcrossterm.${dlext}"
+else
+    install -Dvm 0755 "target/${rust_target}/release/libcrossterm.${dlext}" "${libdir}/libcrossterm.${dlext}"
+fi
+
+install -Dvm 0755 "include/crossterm.h" "${includedir}/crossterm.h"
+
+install_license LICENSE
+"""
+
+platforms = supported_platforms(; experimental=true)
```

Comment:
Experimental is no-op
```suggestion
platforms = supported_platforms()
```